source/plugins/go/src/oms.go

```diff
@@ -1870,6 +1963,16 @@ func InitializePlugin(pluginConfPath string, agentVersion string) {
 	ContainerLogSchemaV2 = false //default is v1 schema
 	ContainerLogV2ConfigMap = (strings.Compare(ContainerLogSchemaVersion, ContainerLogV2SchemaVersion) == 0)
 
+	KubernetesMetadataConfigMap = false
+	KubernetesMetadataConfigMap = (strings.Compare(strings.ToLower(os.Getenv("AZMON_KUBERNETES_METADATA_ENABLED")), "true") == 0)
+	metadataIncludeList := os.Getenv("AZMON_KUBERNETES_METADATA_INCLUDES_FIELDS")
+	Log(fmt.Sprintf("KubernetesMetadataIncludeList from configmap: %+v\n", metadataIncludeList))
+	if KubernetesMetadataConfigMap && len(metadataIncludeList) > 0 {
+		KubernetesMetadataIncludeList = strings.Split(metadataIncludeList, ",")
+	} else {
+		KubernetesMetadataIncludeList = []string{"podLabels", "podAnnotations", "podUid", "image"}
```

Comment:
we can set this as default value and override it if the if condition is true.

Reply:
changed and add condition to handle        [log_collection_settings.metadata_collection]
          enabled = true
          include_fields = [""] scenario